guide/payments/send.md

```diff
@@ -77,6 +77,9 @@ imagesErrors:
     - file: generic-error-details
       alt: A screen showing error details
       caption: Access to error details for problem-solving.
+    - file: offline-error
+      alt: A screen showing error details
```

Comment:
How would the send message button work?

Reply:
"Send message" would open the native share sheet with a pre-written message like "I want to send you some bitcoin. Please open your wallet application to receive them." The user could just pick the recipient and send it off.

Reply:
Makes sense :)